### PR TITLE
Adds salt to chips and jerky

### PR DESCRIPTION
**Pull request by Mickyan**
Chips and jerky from food vendors now contain salt

This is my first PR pls no bully

:cl: Mickyan
tweak: Potato Chips and Beef Jerky from food vendors now contain salt.
/:cl:

[why]: despite being lower than sugar on the resistance scale for virus cures, it's nearly impossible to acquire salt without access to either the kitchen (with its limited supply) or chemistry. This helps balance it out just a little bit.

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_vend.dm [hunk 1]
```diff
@@ -36,11 +36,11 @@
 	icon_state = "chips"
 	trash = /obj/item/trash/chips
 	bitesize = 1
-	list_reagents = list("nutriment" = 1, "sugar" = 3)
+	list_reagents = list("nutriment" = 1, "sugar" = 3, "sodiumchloride" = 1)
 	junkiness = 20
 	filling_color = "#FFD700"
 	tastes = list("salt" = 1, "crisps" = 1)
-	foodtype = JUNKFOOD | FRIED
+	foodtype = JUNKFOOD | VEGETABLES | FRIED
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/food/snacks_vend.dm]: JUNKFOOD takes the place of VEGETABLES in this case. That's more for healthy plant food

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/food/snacks_vend.dm (hunk 1 above)
Comment (Mickyan) [reply code/modules/food_and_drinks/food/snacks_vend.dm]: Fair enough

**Comment by MrDoomBringer** [comment]
welcome to ~where dreams go to die~ the codebase

**Comment by Tlaltecuhtli** [comment]
remove the sugar why the fuck is sugar on my chips

**Comment by Mickyan** [comment]
Sugar is basically a stand-in for inefficient nutriment, to make junk food less appealing compared to the chef's cooking. We could probably come up with a better name for it, but it works.

**Comment by ninjanomnom** [comment]
Look at an ingredients list irl sometime.

**Comment by DaxDupont** [comment]
yeah sugar is often used on paprika crisps.

**Review by Cyberboss — APPROVED** [review]

**Comment by improvedname** [comment]
>99% of the made virusses require salt to cure powercreep pr change first tier virogly cure atleast

**Comment by improvedname** [comment]
Litteraly food item that contains 2 cures for virogly made virruses

**Comment by improvedname** [comment]
If anything this doesnt change balance by a bit but big

As the common virogly virus requires salt/sugar those are also usually the mkst effective ones a clever viroglist would get rid of the salt frlm kitchen then chemistry but why even bother when theres salt and sugar in the local vendor lol

**Comment by improvedname** [comment]
You want salt?

Go to botany.
Go to chemistry
Go to cargo
Go to hydronics
Go to engineering(chemistry circuits)

Salt is the most common thing your just the laziest fuck

**Comment by DaxDupont** [comment]
You want salt?

Just look at @improvedname's comments

**Comment by Aloraydrel** [comment]
Someone unironically wrote that

**Comment by newfren** [comment]
if your death disease is cured by salt you are a bad virologist

**Comment by 81Denton** [comment]
>Salt is the most common thing your just the laziest fuck

Salt is harder to acquire than plenty of the higher tier cures lmao

**Comment by Tlaltecuhtli** [comment]
salt shakers in the kitchen no one is gonna steal those ever

**Review by Cyberboss — APPROVED** [review]
If this is really that much of a balance concern someone else can buff the amount of salt required to cure viruses

**Comment by Alomga** [comment]
> if your death disease is cured by salt you are a bad virologist

Nothing more to add, it's not hard to make a good and hard to cure virus, even a healing virus can be made so it's not cured by sugar/salt/orange juice and is effective.

**Comment by improvedname** [comment]
the meta will just be eat beef jerky and cover 2 viruses cures that is the most common ones too

**Comment by improvedname** [comment]
@Mickyan a hard of cure virus is a joke as the cures are a set table and you already check off two the list after eating beef jerky because virruses are already easy enough to meta? IM COUGHING? MUST BE INFECTED MIGHT EAT BEEF JERKY IF IM ON MY WAY TO MEDBAY ANYWAYS SINCE THAT COVERS 2 CURES 🤖

**Comment by Alomga** [comment]
If salt and sugar are most common cures then those viruses deserve to be cured, let's just merg it, it really doesn't change the balance.

**Comment by epochayur** [comment]
imagine being this mad over video game beef jerky

**Comment by 81Denton** [comment]
@improvedname maybe you should uhhhhhhhhhh remove the salt from your attitude instead

**Comment by improvedname** [comment]
IM SO SALTY I CURED MY OWN HEALVIRUS

**Comment by BlueNothing** [comment]
Imagine being this salty about salt.

**Pull request merged**